pkg/certmanager/cert_manager.go

```diff
@@ -0,0 +1,54 @@
+package certmanager
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-logr/logr"
+	"github.com/manifestival/manifestival"
+	crdclientv1 "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset/typed/apiextensions/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/shipwright-io/operator/pkg/common"
+)
+
+func ReconcileCertManager(ctx context.Context, crdClient crdclientv1.ApiextensionsV1Interface, client client.Client, logger logr.Logger, namespace string) (bool, error) {
+	certificatesInstalled, err := isCertificatesInstalled(ctx, crdClient)
+	if err != nil {
+		return true, err
+	}
+	if !certificatesInstalled {
+		certManagerOperatorInstalled, err := isCertManagerOperatorInstalled(ctx, crdClient)
+		if err != nil {
+			return true, err
+		}
+		if !certManagerOperatorInstalled {
+			return false, fmt.Errorf("cert-manager operator not installed")
+		}
+	}
+
+	manifest, err := common.SetupManifestival(client, "certificates.yaml", logger)
```

Comment:
Not blocking - noticing that this setup code is being run through every reconcile loop. Follow up PRs can refactor things a bit so we only do this setup once. Perhaps use a separate controller?